### DISCUSSION TITLE
Can't install Rocket Chat from Google Play

### DISCUSSION THREAD

**Discussion by titomiky** [RocketChat/Rocket.Chat.ReactNative#3595] [Q&A] [answered]
Hello,
   I'm trying to install Rocket.Chat from Google Play but I can't install it. I have a Xiaomi with Android 11.
Any idea?
Thank you
![Screenshot_2022-01-11-11-45-19-138_com android vending](https://user-images.githubusercontent.com/6685717/148929082-deb2f1d9-dfb5-4952-8a55-d5dc433995e2.jpg)
!

**Comment by dnlsilva** [comment]
Is it just the rocketchat app?
Have you tried installing other apps?

**Comment by titomiky** [comment]
Just rocketchat app

**Reply by dnlsilva** [reply]
What exactly is your device?
Ok it's a Xiaomi, but which model?

**Comment by titomiky** [comment]
Attach model screenshot,
Thank you
![Screenshot_2022-01-11-13-36-53-181_com android settings](https://user-images.githubusercontent.com/6685717/148968318-d10937e1-9e79-42cc-ab90-97ce92d9dff8.jpg)

**Comment by diegolmello** [comment, marked as answer]
Can you read and try this?
https://www.reddit.com/r/PocoPhones/comments/pqxqwu/poco_x3_nfc_dotos_513_cant_install_some_apps_on/
https://www.hardreset.info/devices/poco/poco-x3-nfc/faq/tips-tricks/google-play-errors/
https://c.mi.com/thread-3532281-1-1.html

**Comment by titomiky** [comment]
Thank you but I followed similar posts that I found googling with no success.
I have different LG mobile phones and Samsung with android 10 and 11, and I only could install in a Samsung A12 with Android 11.
I really don't understand...

**Reply by titomiky** [reply]
Problem solved. 
We were trying to install similar apps.
Thank you!
